condor_watch_q.py

```diff
@@ -462,9 +472,10 @@ def table_by_cluster_id(self):
             for cluster_id, proc_statuses in state.items():
                 d = {js: 0 for js in JobStatus}
                 live_job_ids = []
+             
                 for proc_status in proc_statuses.values():
                     d[proc_status] += 1
-
+                                  
```

Comment:
Please avoid unnecessary churn - particularly, here where you put in empty spaces.